src/NServiceBus.Core/Recoverability/Faults/MoveFaultsToErrorQueueBehavior.cs

```diff
@@ -71,12 +79,26 @@ public MoveFaultsToErrorQueueBehavior(CriticalError criticalError, IPipelineBase
         HostInformation hostInformation;
         BusNotifications notifications;
         string errorQueueAddress;
+        string localAddress;
         static ILog Logger = LogManager.GetLogger<MoveFaultsToErrorQueueBehavior>();
 
         public class Registration : RegisterStep
         {
-            public Registration()
-                : base("MoveFaultsToErrorQueue", typeof(MoveFaultsToErrorQueueBehavior), "Moved failing messages to the configured error queue")
+            public Registration(ReadOnlySettings settings, string localAddress)
+                : base("MoveFaultsToErrorQueue", typeof(MoveFaultsToErrorQueueBehavior), "Moved failing messages to the configured error queue", b =>
+                {
```

Comment:
I put it here only because currently satellites reuse this class. Otherwise this piece of code should live in the proper feature. When we finally clean up satellites there will be no need for this funky code so I vote against introducing an abstract method for it.

@andreasohlund as I said, the `RegisterStep` is here only because of reuse in three different places. Otherwise I'd just use `Register` in the feature itself.